INFO - 2023-12-16 17:12:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-16 17:12:10 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(174): mysqli->real_connect()
#1 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 C:\xampp\htdocs\catcare\app\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\Home->index()
#8 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
#9 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\xampp\htdocs\catcare\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 C:\xampp\htdocs\catcare\app\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\Home->index()
#7 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
#8 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#9 C:\xampp\htdocs\catcare\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-12-16 17:12:10 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\catcare\\public\\index.php')
INFO - 2023-12-16 17:12:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-16 17:12:34 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(174): mysqli->real_connect()
#1 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 C:\xampp\htdocs\catcare\app\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\Home->index()
#8 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
#9 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\xampp\htdocs\catcare\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 C:\xampp\htdocs\catcare\app\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\Home->index()
#7 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
#8 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#9 C:\xampp\htdocs\catcare\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-12-16 17:12:34 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\catcare\\public\\index.php')
INFO - 2023-12-16 17:12:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-16 17:12:48 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(174): mysqli->real_connect()
#1 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 C:\xampp\htdocs\catcare\app\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\Home->index()
#8 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
#9 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\xampp\htdocs\catcare\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 C:\xampp\htdocs\catcare\app\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\Home->index()
#7 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
#8 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#9 C:\xampp\htdocs\catcare\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\catcare\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-12-16 17:12:48 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH\Controllers\Home.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\catcare\\public\\index.php')
INFO - 2023-12-16 17:13:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-16 17:13:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
